hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -237,6 +237,8 @@ public enum Units { TB, GB, MB, KB, B }
 
   public static final int INVALID_PORT = -1;
 
+  public static final long OBJECT_ID_DEFAULT = 0L;
```

Comment:
Make the variable name less generic. Maybe `OBJECT_ID_RECLAIM_BLOCKS`

And put a comment there to clearly describe the intention of the constant.